### DISCUSSION TITLE
Should the vim formula depend on libsodium?

### DISCUSSION THREAD

**Discussion by wbpayne22902** [Homebrew/discussions#3662] [Tap maintenance and Homebrew development] [answered]
I was wondering if it might be possible to request that the vim formula depend on libsodium so that the vim bottles have the sodium feature.  Currently if you build from HEAD after libsodium is installed vim builds in support for libsodium but if vim is installed from a bottle there is no libsodium support.
I use the xchacha2 cryptmethod with vim which requires the sodium feature to edit encrypted text files and this would be useful so I could avoid building vim from HEAD just to use the sodium feature.

**Comment by gromgit** [comment, marked as answer]
> Should the vim formula depend on libsodium?

Only if someone's willing to:
1. raise a PR with the necessary changes in the formula, and
2. actually _test_ the resulting build to ensure that it correctly incorporates the library _and_ exposes the desired new functionality

**Reply by wbpayne22902** [reply]
Done.  I modified the formula and ran all tests and put in a PR as requested.

homebrew-core pr #110966
